packages/astro/package.json

```diff
@@ -100,7 +100,7 @@
     "test:e2e:match": "playwright test -g"
   },
   "dependencies": {
-    "@astrojs/compiler": "^0.29.15",
+    "@astrojs/compiler": "next--module-id",
```

Comment:
Note that there is an associated compiler change. This should not be merged until that change is in.